src/ObjectStore.js

```diff
@@ -422,26 +431,22 @@ function getLatest(id: Id) {
   return store[id] ? deepFetch(id) : null;
 }
 
-var ObjectStore: { [key: string]: any } = {
-  storeObject: storeObject,
-  removeObject: removeObject,
-  addSubscriber: addSubscriber,
-  removeSubscriber: removeSubscriber,
-  fetchSubscribers: fetchSubscribers,
-  stackMutation: stackMutation,
-  destroyMutationStack: destroyMutationStack,
-  resolveMutation: resolveMutation,
-  commitDelta: commitDelta,
-  storeQueryResults: storeQueryResults,
-  getDataForIds: getDataForIds,
-  deepFetch: deepFetch,
-  getLatest: getLatest
-};
+module.exports.storeObject = storeObject;
```

Comment:
Sadly Flow doesn't like mixing `export` (ES6) and `module.exports` (CommonJS), and ES6 modules don't allow conditional exports, it seems...